app/components/shared/filters_component/filters_component.html.haml

```diff
@@ -0,0 +1,61 @@
+.filters
+  .filters__heading.moj-filter
+    %div{ class: "#{@options[:close_all] ? 'filters__heading-container' : 'moj-filter__header'}" }
```

Comment:
Now we're using `attr_accessor` in the component, we should be able to access all of the variables (don't need to access the instance variable)